### ISSUE TITLE
Psychopy/pavlovia: cannot read property '_gui' of undefined

### ISSUE THREAD

**Issue by cmoreau5** [psychopy/psychopy#3095]
I downloaded the newest update of psychopy because of the issue with anything prefixed with $ breaking once exported as HTML. Once I sync the project on pavlovia, and try to run it online, I get a new error pop up on psychopy:

`wx._core.wxAssertionError: C++ assertion "(argtype & (wxFormatStringSpecifier<T>::value)) == argtype" failed at C:\PROJECTS\bb2\dist-win64-py36\build\ext\wxWidgets\include\wx/strvararg.h(449) in wxArgNormalizer<unsigned __int64>::wxArgNormalizer(): format specifier doesn't match argument type`

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "_ctypes/callbacks.c", line 234, in 'calling callback function'
  File "C:\Program Files\PsychoPy3\lib\site-packages\pyglet\window\win32\__init__.py", line 635, in f
    event_handler = event_handlers.get(msg, None)
SystemError: <built-in method get of dict object at 0x000002AAE106F360> returned a result with an error set

The following also pops up on pavlovia:
![image](https://user-images.githubusercontent.com/70174690/91097268-5f98fd00-e62d-11ea-812b-d0b13f4d7ca9.png)

**Comment by OliJimbo** [comment]
This happens when you define a variable twice - it’s likely that in your psychojs code component there is a variable that is declared also in the main .js file - take a look and delete it if it’s there!

**Comment by OliJimbo** [comment]
I was wrong - it's because of code in the text and graphic components - bear with me - I'll see if there's work around

**Comment by peircej** [comment]
Very hard to know from this whether there's a problem with a) the experiment (in which case better to discuss on the forum) or b) a bug in psychopy or c) a bug with the JS library

Could you provide a link or upload the experiment file so we can rule out (a)?
 Could you tell us what version of PsychoPy you're using so we can check (b)?

**Comment by cmoreau5** [comment]
I'm still unsure what the issue was. The error kept popping up every 30 or so seconds even when I wasn't doing anything on Psychopy. I'm thinking something likely went wrong with the download. I ended up uninstalling the 2020.2.3 version of psychopy and re-installing the 2020.2.2 version and it works fine. I also see now that there's a 2020.2.4 version, which dealt with some other issues I also encountered. Thanks!
